CONTRIBUTING.md

```diff
@@ -2,107 +2,71 @@
 
 We're not ready for contributions yet, this document exists as a guide for those working on govuk-frontend.
 
-## Code of Conduct
-Please read [CODE_OF_CONDUCT.md](./CODE_OF_CONDUCT.md) before contributing.
-
-## Running locally
-
-You'll need [Git](https://help.github.com/articles/set-up-git/) and [Node.js](https://nodejs.org/en/) installed to get this project running.
+## Contents of this file
+
+### For contributors
+- [Code of conduct](#code-of-conduct)
+- [Application architecture](#application-architecture)
+- [Running locally](#running-locally)
+- [Conventions to follow](#conventions-to-follow)
+ - [Indentation and whitespace](#indentation-and-whitespace)
```

Comment:
If you use two spaces to indent this will actually render as a sub-list. At the minute it appears as one long list with no indentation.

- List item
  - Sub list-item

```
- List item
  - Sub list-item
```

- List item
 - Sub list item

```
- List item
 - Sub list item
```